pom.xml

```diff
@@ -1479,16 +1479,6 @@
 
   <repositories>
     <repository>
-      <id>jboss-snapshots</id>
-      <url>https://repository.jboss.org/nexus/content/repositories/snapshots</url>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-    </repository>
-    <repository>
```

Comment:
We need Sonatype repo for the Indy snapshots. We don't need the JBoss one as we build all our components. Sometimes it was happening to me, that I wanted to run tests for e.g. rest and Maven started downloading PNC snapshots from the repo instead of using the local one.
Do you think we need the JBoss snapshots repo?